src/components/App/index.js

```diff
@@ -14,9 +16,17 @@ class App extends Component {
         <Fragment>
           <Navbar />
           <Main>
-            <Route path={'/posts'} component={ListPage} />
-            <Route path={'/router'} component={RouterPage} />
-            <Route exact path="/" component={() => <Redirect to="/posts" />} />
+            <Switch>
+              <Route path={'/posts'} component={PostListPage} />
```

Comment:
Just a nitpick: curly brackets not needed here and in few lines below

```suggestion
              <Route path='/posts' component={PostListPage} />
```